src/Components/CollectionCard/CollectionCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import imgIcon from '../../assets/icons/image.png';
+import starredIcon from '../../assets/icons/Star.png';
+
+function CollectionCard({ card }) {
+  return (
+  <div className={`bg-${card.type.color}-200 w-80 m-2 p-2 shadow-2xl rounded-xl`}>
+       <p className="text-4xl text-center font-bold">{card.date}</p>
+       <p className="text-2xl text-center font-light">{card.type.name}</p>
```

Comment:
how can you get the `card.type.name` directly, isn't the card.type just an `id`? like this
```jsx 
card {
   type: {
   id: 5
   }
}
```

Reply:
inside the type there is a name:
card {
   type: {
   id: 5,
color:...,
name:...
   }
}

Reply:
if that's the case then you don't need the collections array, right?

Reply:
yeah I think you are right but the collection array was built only to specify for each type a color. If you think it is not needed then we can delete it. I think we can loop over it when creating a card, b y checking the name and assigning the corresponding color instead of using many if else or switch case. just tell me what you find more convenient.